core/services/chainlink/application.go

```diff
@@ -160,6 +160,17 @@ func NewApplication(opts ApplicationOpts) (Application, error) {
 	eventBroadcaster := opts.EventBroadcaster
 	externalInitiatorManager := opts.ExternalInitiatorManager
 
+	if cfg.AutoPprofEnabled() {
+		globalLogger.Info("Nurse service (automatic pprof profiling) is enabled")
+		nurse := health.NewNurse(cfg, globalLogger)
+		err := nurse.Start()
```

Comment:
Should `nurse` be passed along as a `service.Service` to `ChainlinkApplication.subservices` so it is `Close()`ed gracefully?

Reply:
(or perhaps it should be treated special and e.g. closed last so that we capture shutdown as well? :shrug:)